packages/osc-ecommerce/app/components/Cards/BlogCard.tsx

```diff
@@ -50,7 +47,7 @@ export const BlogCard = (props: Props) => {
                         overlayColor={
                             heroData?.image?.imageStyles?.overlayColor
                                 ? heroData?.image?.imageStyles?.overlayColor
-                                : theme?.color
+                                : data.reference.theme?.color
```

Comment:
We're adding optional chaining to a few places on `data.` but not here or on line 63. Should we?
```suggestion
                                : data?.reference.theme?.color
```

Reply:
Good shout. Updated.